frb_rust/src/ffi/web.rs

```diff
@@ -405,3 +408,65 @@ mod tests {
         assert_eq!(ns, 567_000_000);
     }
 }
+
+#[derive(Debug)]
+pub struct DartOpaque {
+    /// Dart object ptr
+    handle: *mut JsValue,
```

Comment:
You can make `Box<JsValue>`, but in any case it will be about the same. This is because `JsValue` is not a `Send` (we are delegating the drop to Dart).

Reply:
Hmm, if it is not Send, is it safe we unsafely force it to be Send?

Reply:
Yes, we only use how to give back Dart or unwrap on a Dart thread

Reply:
Great! Then maybe we can add a comment there.

Btw, I personally prefer `Box` over `*mut` because the former is higher level and looks safer. So is it possible we migrate to it?
(I have discussed the possibility of `struct DartOpaque { inner: Option<DartOpaqueInner> }`, so double-free is not a problem as long as we properly set Option to None

Reply:
Yes, I already refactored